packages/dev/core/src/FlowGraph/flowGraphDataConnection.ts

```diff
@@ -11,12 +10,12 @@ import type { ValueContainer } from "./valueContainer";
  * An output point will only have a value if it is connected to an input point. Furthermore,
  * if the point belongs to a "function" node, the node will run its function to update the value.
  */
-export class FlowGraphDataConnectionPoint<T> {
-    private _connectedPoint: Nullable<FlowGraphDataConnectionPoint<T>>;
+export class FlowGraphDataConnection<T> {
+    private _connectedPoint: Nullable<FlowGraphDataConnection<T>>;
 
-    constructor(public name: string, public role: FlowGraphConnectionPointRole, private _ownerBlock: FlowGraphBlock, private _valueContainer: ValueContainer<T>) {}
+    constructor(public name: string, public role: FlowGraphConnectionType, private _ownerBlock: FlowGraphBlock, private _valueContainer: ValueContainer<T>) {}

```

Comment:
Still have constructors without an access modifier. I recommend adding it because the default is public and adding it forces us to at least think about whether it should be public or not.